packages/strapi-design-system/src/SubNav/SubNav.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { Grid } from '../Grid';
+
+const SubNavWrapper = styled(Grid)`
+  width: ${232 / 16}rem;
+  background: ${({ theme }) => theme.colors.neutral100};
+  height: 100%;
+  position: relative;
+  overflow-y: auto;
+`;
+
+export const SubNav = ({ condensed, ariaLabel, ...props }) => {
+  return (
+    <SubNavWrapper aria-label={ariaLabel} as="nav" rows="auto auto 1fr auto auto" condensed={condensed} {...props} />
```

Comment:
The rows props does not exist anymore on the Grid component, this should not work 🤔

Reply:
It is useless 👍🏽 . I'll remove it.

Reply:
Same in the MainNav